arches/app/media/js/views/components/search/resource-type-filter.js

```diff
@@ -24,6 +24,9 @@ define([
                     self.resourceModels.push(res);
                     //}
                 });
+                self.resourceModels.sort(function(a,b) {
+                    return a.name.toLowerCase().localeCompare(b.name.toLowerCase());
+                });  // sort resource model list alphabetically
```

Comment:
No need to make any changes, but I think your code explains itself really well here, so your comment isn't really necessary.